src/common/KokkosKernels_BitUtils.hpp

```diff
@@ -203,6 +207,36 @@ int pop_count( long long i ){
   return __popcnt8(i);
 }
 
+#elif defined (_MSC_VER)
```

Comment:
Like above, I'm wondering if `_MSC_VER` should be changed to the Kokkos macro `KOKKOS_COMPILER_MSVC` from here:
https://github.com/kokkos/kokkos/blob/98b8daf4a5644c3166123ded7bfa5b4d0264399b/core/src/Kokkos_Macros.hpp#L190

Reply:
Let's change to `KOKKOS_COMPILER_MSVC`